crates/relayer/src/chain/cosmos.rs

```diff
@@ -120,6 +122,8 @@ pub mod types;
 pub mod version;
 pub mod wait;
 
+pub const DEFAULT_GRPC_MAX_MESSAGE_LENGTH: u64 = n_mib_bytes(32);
```

Comment:
Let's remove this and use a default straight in the config struct to avoid the if/else's below